### DISCUSSION TITLE
Why does dbTechLayer.getArea() return a float?

### DISCUSSION THREAD

**Discussion by rovinski** [The-OpenROAD-Project/OpenROAD#6221] [Q&A] [answered]
`dbTechLayer.getArea()` seems to return a float in um^2, whereas `getwidth()` returns dbu. Is there a reason for this? It seems like it would be good to be consistent.

**Comment by maliberty** [comment, marked as answer]
Converted to issue https://github.com/The-OpenROAD-Project/OpenROAD/issues/6222
